R/pkg/R/generics.R

```diff
@@ -1535,9 +1535,7 @@ setGeneric("spark.freqItemsets", function(object) { standardGeneric("spark.freqI
 #' @export
 setGeneric("spark.associationRules", function(object) { standardGeneric("spark.associationRules") })
 
-#' @param object a fitted ML model object.
```

Comment:
I think it makes more sense to keep param annotations with concrete implementation and keeping both would violate style by duplicating Rd entries.